bundle/edu.gemini.itc.shared/src/main/scala/edu/gemini/itc/shared/ConfigExtractor.scala

```diff
@@ -264,7 +266,7 @@ object ConfigExtractor {
         fieldLens <- extract[FieldLens]     (c, AoFieldLensKey)
         wfsMode   <- extract[GuideStarType] (c, AoGuideStarTypeKey)
       } yield {
-        val separation = distance(targetEnv.getBase, guideStar, when)
+        val separation = distance(targetEnv.getAsterism, guideStar.getTarget, when)
```

Comment:
I'm inclined to leave this as-is for now. I will add a comment with a link to this conversation so we can review later.